apps/users/models.py

```diff
@@ -129,42 +99,37 @@ def profile_picture(instance, filename):
 
 #====
 class User(AbstractBaseUser):
-    GENDER_CHOICES = [
-        ('Male', 'Male'),
-        ('Female', 'Female'),
-        ('Other', 'Other'),
-        ('Prefer Not to Say', 'Prefer Not to Say')
-    ]
+    GENDER_CHOICES = [('Male', 'Male'),('Female', 'Female'),('Other', 'Other'),('Prefer Not to Say', 'Prefer Not to Say')]
+    TITLE_CHOICES = [('Mr', 'Mr'),('Miss', 'Miss'),('Mrs', 'Mrs'),('Ms', 'Ms'),('Mx', 'Mx')]
```

Comment:
Use only Mr. and Ms. and default is None

Reply:
Done